README.rst

```diff
@@ -59,6 +59,36 @@ Devstack Set Up
 
 You can use the make targets defined in the ``Makefile`` to interact with the running ``edx-exams`` Docker containers.
 
+LTI Configuration for Local Development
```

Comment:
I'd say so. Since this is meant to be seen by Open Edx's community, I believe it makes sense for this public README to be the source of truth instead of our internal doc. I'll make sure to do that link once this is merged.